SPECS/cloud-hypervisor/cloud-hypervisor.spec

```diff
@@ -19,9 +19,14 @@ Source0:        https://github.com/cloud-hypervisor/cloud-hypervisor/archive/ref
 #   cd %{name}-%{version}
 #   cargo vendor > config.toml
 #   tar -czf %{name}-%{version}-cargo.tar.gz vendor/
+# rename the tarball to %{name}-%{version}-cargo.tar.gz when updating version
 Source1:        %{name}-%{version}-cargo.tar.gz
 Source2:        config.toml
 Patch0:         CVE-2023-45853.patch
+Patch1:         CVE-2023-50711-vmm-sys-util-fix1.patch 
+Patch2:         CVE-2023-50711-vmm-sys-util-fix2-cargo.patch
+Patch3:         CVE-2023-50711-vhost.patch
+Patch4:         CVE-2023-50711-versionize.patch
```

Comment:
@nisamson, will a patch file `CVE-XXXX-YYYY-something.patch` be also interpreted correctly by our CVE scanning logic?

Reply:
Yes, the patch is interpreted correctly.